gulpfile.babel.js

```diff
@@ -292,83 +363,115 @@ gulp.task('rpm:linux', ['package:linux'], (done) => {
   });
 });
 
+gulp.task('rpm:linux:32', ['package:linux:32'], (done) => {
+  const redhat = require('electron-installer-redhat');
+
+  redhat(_.extend({}, redhatDefaults, {
+    src: `dist/${packageJSON.productName}-linux-ia32`,
+    arch: 'i386',
+  }), (err) => {
+    console.log('32bit rpm package built');
+    if (err) return done(err);
+    done();
+  });
+});
+
+gulp.task('rpm:linux:64', ['package:linux:64'], (done) => {
+  const redhat = require('electron-installer-redhat');
+
+  redhat(_.extend({}, redhatDefaults, {
+    src: `dist/${packageJSON.productName}-linux-x64`,
+    arch: 'amd64',
+  }), (err) => {
+    console.log('32bit rpm package built');
+    if (err) return done(err);
+    done();
+  });
+});
+
 gulp.task('make:linux', ['deb:linux', 'rpm:linux'], (done) => {
```

Comment:
Err, that's existing code used to build via circle CI, unmodified. Did you mean another line? I didn't  do anything that would mess with the normal CI process this time. Only added the 6 Linux only build commands.